### PR TITLE
Add getPhotoSettings

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Following https://www.chromestatus.com/feature/4843864737185792, we need to update the article.

Please @yellowdoge review.

Note that "ideal" code snippet doesn't work and says `PhotoCapabilities` are undefined, do you know why?

```js
var widthSlider = document.querySelector('input[type=range]');
// ...
Promise.all([imageCapture.getPhotoCapabilities(), imageCapture.getPhotoSettings()])
  .then(function(photoCapabilities, photoSettings) {
    widthSlider.min = photoCapabilities.imageWidth.min;
    widthSlider.max = photoCapabilities.imageWidth.max;
    widthSlider.step = photoCapabilities.imageWidth.step;
    widthSlider.value = photoSettings.imageWidth;
  })
  .catch(error => console.error('getPhotoCapabilities() error:', error));
```

Moreover `redEyeReduction` and `fillLightMode` never show up for me in Android and Chrome OS. Is that expected?

PS: I'm also updating our Photo Resolution sample at https://github.com/GoogleChrome/samples/pull/530

### CODE REVIEW

File: src/content/en/updates/2016/12/imagecapture.md [hunk 1]
```diff
@@ -122,10 +123,13 @@ streaming, so make sure there is a delay between calling these methods and
 
 "Non-Live" camera capabilities and settings are manipulated via the
 `ImageCapture` object: `ImageCapture.getPhotoCapabilities()` returns a
-[`PhotoCapabilities`](https://w3c.github.io/mediacapture-image/##photocapabilities-section)
-object that provides access to non live available camera options. The photo
-resolution, red eye reduction and flash mode (except torch) belong this section,
-for example:
+[`PhotoCapabilities`](https://w3c.github.io/mediacapture-image/#photocapabilities)
+object that provides access to non live available camera options.
```
Comment (jpmedley) [inline src/content/en/updates/2016/12/imagecapture.md]: 'non live available'. What does this mean?

**Comment by petele** [comment]
@beaufortfrancois - would you please fix the merge conflict?

**Comment by petele** [comment]
@beaufortfrancois - would you please fix the merge conflicts then happy to merge and push live.

File: src/content/en/updates/2016/12/imagecapture.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ project_path: /web/_project.yaml
 book_path: /web/updates/_book.yaml
 description: Image Capture is an API to control camera settings and take photos.
 
-{# wf_updated_on: 2017-05-25 #}
+{# wf_updated_on: 2017-07-29 #}
 {# wf_published_on: 2016-12-05 #}
 {# wf_tags: canvas,chrome56,media,webrtc #}
 {# wf_featured_image: /web/updates/images/2016/12/imagecapture/featured.jpg #}
```
Comment (ebidel) [inline src/content/en/updates/2016/12/imagecapture.md]: please add `{# wf_blink_components: Blink>ImageCapture #}`

Comment (beaufortfrancois) [inline src/content/en/updates/2016/12/imagecapture.md]: Done.

File: src/content/en/updates/2016/12/imagecapture.md (hunk 1 above)
Comment (beaufortfrancois) [inline src/content/en/updates/2016/12/imagecapture.md]: I've rephrased it to `"Non-Live" available camera options`

**Comment by beaufortfrancois** [comment]
@petele I'm good now. Thanks for your patience!

**Comment by beaufortfrancois** [comment]
@yellowdoge May you have a look at https://github.com/google/WebFundamentals/pull/4890#issue-246499858?

**Comment by beaufortfrancois** [comment]
Preview is available at https://pr-4890-dot-web-central.appspot.com/web/updates/2016/12/imagecapture

**Pull request merged**